extension/scripts/features/oc2-filter/ttOC2Filter.js

```diff
@@ -119,4 +152,63 @@
 		removeContainer("OC Filter");
 		document.findAll(".tt-oc2-list .tt-hidden").forEach((x) => x.classList.remove("tt-hidden"));
 	}
+
+	// Helper function to determine if we're on the completed crimes tab
+	function isCompletedCrimesTab() {
+		// Check if we're viewing completed crimes vs recruiting or planning
+		const activeTab = document.querySelector("#faction-crimes-root [class*='buttonsContainer___'] > [class*='active___']");
+		
+		if (!activeTab) {
+			return false; // Default to false if we can't determine tab
+		}
+		
+		const tabText = activeTab.textContent.trim().toLowerCase();
+		
+		// Check if the active tab is NOT recruiting or planning (so it must be completed)
+		return !tabText.includes('recruiting') && !tabText.includes('planning');

```

Comment:
I would prefer the check to just be on "completed" instead. In the off chance that a new tab is introduces, I would like not to have to update this.